variables.tf

```diff
@@ -1,11 +1,154 @@
-##############################################################################
-# Input Variables
-##############################################################################
+variable "resource_group_id" {
+  description = "ID of resource group to use when creating the event stream instance"
+  type        = string
+}
 
-#variable "myvariable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = ""
-#}
+variable "es_name" {
+  type        = string
+  description = "Name of the resource instance"
```

Comment:
perhaps be more specific - The name to give the IBM Event Streams instance created by this module